File: library/src/main/java/com/ciandt/dragonfly/infrastructure/PermissionsMapping.java
package com.ciandt.dragonfly.infrastructure;

import android.Manifest;

import java.util.Collections;
import java.util.List;

/**
 * Created by iluz on 6/12/17.
 */

public interface PermissionsMapping {

    List<String> CAPTURE_FRAME = Collections.singletonList(Manifest.permission.WRITE_EXTERNAL_STORAGE);

    List<String> SAVE_IMAGE_TO_GALLERY = Collections.singletonList(Manifest.permission.WRITE_EXTERNAL_STORAGE);
}

File: app/src/main/java/com/ciandt/dragonfly/example/config/PermissionsMapping.kt
package com.ciandt.dragonfly.example.config

import android.Manifest

/**
 * Created by iluz on 7/10/17.
 */
interface PermissionsMapping {
    companion object {
        val SAVE_IMAGE_TO_GALLERY = listOf(Manifest.permission.WRITE_EXTERNAL_STORAGE)
        val REAL_TIME: String = Manifest.permission.CAMERA
    }
}